docs/development-update-22.44.md

```diff
@@ -0,0 +1,120 @@
+# Development Update 22.44
+
+## Overview
+
+This is an interim update ("code drop") to give the validation team
+an opportunity to take the DPDK code out for a test drive.
+
+## Project Name
+
+Note that the P4-OVS Split Architecture has a new name: **P4 Control Plane**.
+
+## What's Changed
+
+### Highlights
+
+- Restructured Stratum to support TDI variants
+- Integrated DPDK support from P4-OVS
+- Disabled unsupported port attributes in DPDK
+- Updated OVS to newer release
+- Replace `gnmi_cli` with `gnmi-ctl` in DPDK build
+
+### TDI variants
+
+When we compared the Stratum files in P4-OVS with their Tofino equivalents,
+we found significant changes in the ChassisManager, Hal, Main, and Switch
+classes, as well as SdeWrapper and YangParseTreePaths.
+
+We addressed the first four classes by creating *dpdk* and *tofino*
+subdirectories under the *tdi* directories, and giving each target its
+own implementation of the class.
+
+For SdeWrapper, we refactored `tdi_sde_wrapper.cc` (\~3800 lines) into
+multiple, smaller source files, isolating the target-specific methods
+in a single file. We then created separate versions of this file for
+the DPDK and Tofino targets, and moved them to the target-specific
+subdirectories
+
+For YangParseTreePaths, we refactored `yang_parse_tree_paths.cc`
+(originally \~4100 lines, increasing to \~5600 lines in P4-OVS) into
+multiple, smaller source files, separating out sections of code that
+had been heavily modified. We then created alternate versions for DPDK.
+See `refactoring_yang_parse_tree_paths.md` in the `stratum/docs`
```

Comment:
Maybe link the doc instead of just specifying the name?

Reply:
Definitely, now that I know I can.

Reply:
Works like a champ. 🥇